### PR TITLE
setPantosToken to be super critical ops and pantos token to have super critical ops as owner

### PR DESCRIPTION
**Pull request by chandraprakash**
## What type of PR is this? (check all applicable)

- [X] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Pantos token to have super critical ops as owner
setPantosToken() to be super critical ops
## Related Tickets & Documents

- Related Issue #
- Closes #

## QA Instructions

_Please replace this line with instructions on how to test your changes._

## [optional] Are there any post deployment tasks we need to perform?

### CODE REVIEW

File: test/PantosHub.t.sol [hunk 1]
```diff
@@ -1253,14 +1496,64 @@ contract PantosHubTest is PantosHubDeployer {
...
+
+        PantosTypes.ExternalTokenRecord
+            memory externalTokenRecord = pantosHubProxy.getExternalTokenRecord(
+                PANDAS_TOKEN_ADDRESS,
+                uint256(otherBlockchain.blockchainId)
+            );
+        assertFalse(externalTokenRecord.active);
+    }
+
+    function test_unregisterExternalTokenBySuperCriticalOpsAndPaused()
```
Comment (markuslevonyak) [inline test/PantosHub.t.sol:1542]: Our naming convention would be `test_unregisterExternalToken_BySuperCriticalOpsAndPaused`.

File: test/PantosHub.t.sol [hunk 2]
```diff
@@ -1107,46 +1272,92 @@ contract PantosHubTest is PantosHubDeployer {
...
+                uint256(otherBlockchain.blockchainId)
+            );
+        assertTrue(externalTokenRecord.active);
+        assertEq(
+            externalTokenRecord.externalToken,
+            EXTERNAL_PANDAS_TOKEN_ADDRESS
+        );
+    }
+
+    function test_registerExternalTokenBySuperCriticalOpsAndPaused() external {
```
Comment (markuslevonyak) [inline test/PantosHub.t.sol:1355]: Our naming convention would be `test_registerExternalToken_BySuperCriticalOpsAndPaused`.

File: test/PantosHub.t.sol [hunk 3]
```diff
@@ -1107,46 +1272,92 @@ contract PantosHubTest is PantosHubDeployer {
...
+                uint256(otherBlockchain.blockchainId)
+            );
+        assertTrue(externalTokenRecord.active);
+        assertEq(
+            externalTokenRecord.externalToken,
+            EXTERNAL_PANDAS_TOKEN_ADDRESS
+        );
+    }
+
+    function test_registerExternalTokenBySuperCriticalOps() external {
```
Comment (markuslevonyak) [inline test/PantosHub.t.sol:1333]: Our naming convention would be `test_registerExternalToken_BySuperCriticalOps`.

File: test/PantosHub.t.sol [hunk 4]
```diff
@@ -1060,7 +1111,69 @@ contract PantosHubTest is PantosHubDeployer {
         checkTokenIndices();
     }
 
-    function test_unregisterToken_ByNonDeployerAndPaused() external {
+    function test_unregisterTokenBySupeCriticalOps() external {
```
Comment (markuslevonyak) [inline test/PantosHub.t.sol:1114]: Our naming convention would be `test_unregisterToken_BySuperCriticalOps` (also note the "r" in "Super").

File: test/PantosHub.t.sol [hunk 5]
```diff
@@ -984,11 +986,49 @@ contract PantosHubTest is PantosHubDeployer {
...
+        pantosHubProxy.registerToken(PANDAS_TOKEN_ADDRESS);
+
+        PantosTypes.TokenRecord memory tokenRecord = pantosHubProxy
+            .getTokenRecord(PANDAS_TOKEN_ADDRESS);
+        assertTrue(tokenRecord.active);
+        assertTrue(inArray(PANDAS_TOKEN_ADDRESS, loadPantosHubTokens()));
+        checkTokenIndices();
+    }
+
+    function test_registerTokenBySuperCriticalOpsWhenPaused() external {
```
Comment (markuslevonyak) [inline test/PantosHub.t.sol:1021]: Our naming convention would be `test_registerToken_BySuperCriticalOpsWhenPaused`.

File: test/PantosHub.t.sol [hunk 6]
```diff
@@ -984,11 +986,49 @@ contract PantosHubTest is PantosHubDeployer {
...
+        pantosHubProxy.registerToken(PANDAS_TOKEN_ADDRESS);
+
+        PantosTypes.TokenRecord memory tokenRecord = pantosHubProxy
+            .getTokenRecord(PANDAS_TOKEN_ADDRESS);
+        assertTrue(tokenRecord.active);
+        assertTrue(inArray(PANDAS_TOKEN_ADDRESS, loadPantosHubTokens()));
+        checkTokenIndices();
+    }
+
+    function test_registerTokenBySuperCriticalOps() external {
```
Comment (markuslevonyak) [inline test/PantosHub.t.sol:1004]: Our naming convention would be `test_registerToken_BySuperCriticalOps`.

File: test/PantosHub.t.sol [hunk 7]
```diff
@@ -1060,7 +1111,69 @@ contract PantosHubTest is PantosHubDeployer {
...
+            );
+        address[] memory tokens = pantosHubProxy.getTokens();
+        assertEq(tokens.length, 1);
+        assertEq(tokens[0], PANTOS_TOKEN_ADDRESS);
+        assertFalse(tokenRecord.active);
+        assertFalse(externalTokenRecord.active);
+        checkTokenIndices();
+    }
+
+    function test_unregisterTokenBySupeCriticalOpsAndPaused() external {
```
[the remark this answers is not in the document]
Comment (chandraprakash) [reply test/PantosHub.t.sol]: You have got sharp eyes :)

File: test/PantosHub.t.sol [hunk 8]
```diff
@@ -1060,7 +1111,69 @@ contract PantosHubTest is PantosHubDeployer {
         checkTokenIndices();
     }
 
-    function test_unregisterToken_ByNonDeployerAndPaused() external {
+    function test_unregisterToken_BySupeCriticalOps() external {
```
Comment (markuslevonyak) [inline test/PantosHub.t.sol:1114]: "r" missing in "Super".

**Review by markuslevonyak — APPROVED** [review]

**Review by juanmanuel-tirado — APPROVED** [review]

**Pull request merged**